rpmio/macro.cc

```diff
@@ -1992,10 +1994,15 @@ rpmExpand(const char *arg, ...)
 int
 rpmExpandNumeric(const char *arg)
 {
+    int res;
```

Comment:
Guess I just didn't notice this before, but don't introduce uninitialized variables. Nothing needs `res` until it gets the value from the clamp, so just introduce it there.